### ISSUE TITLE
/anvil dont work for 1.21.1 servers?

### ISSUE THREAD

**Issue by Aryan20569** [IntellectualSites/FastAsyncWorldEdit#3014]
### Server Implementation

Paper

### Server Version

1.21.3

### Describe the bug

I want to unload chunks which are not useful, but /anvil command is not showing up?

### To Reproduce

n/a

### Expected behaviour

should delete unused chunks

### Fawe Debugpaste

n/a

### Fawe Version

latest

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit/ and the issue still persists.

**Comment by ShayBox** [comment]
Any progress on this?

**Comment by MattBDev** [comment]
> Any progress on this?

No. I'm interested in this functionality so maybe something will happen by the end of the year. I can't give an ETA though.

**Comment by itzzjustmateo** [comment]
what for an anvil feature?
